blueoil/cmd/output_event.py

```diff
@@ -63,32 +74,42 @@ def output(tensorboard_dir, output_dir, metrics_keys, steps, output_file_base="m
 
     columns = [_column_name(event_accumulator, metrics_key)
                for event_accumulator, metrics_key in itertools.product(event_accumulators, metrics_keys)]
-    columns.sort()
-    df = pd.DataFrame([], columns=columns)
+
+    values_step_dict = {}
```

Comment:
Do we need this initialize code?

Reply:
It is not necessary. I will remove this.
Thank you for pointing it out.